CONTRIBUTING.md

```diff
@@ -0,0 +1,300 @@
+scope of change, capacity, etc. At least one reviewer is assigned to coordinate
+with the Owner as the PEP moves through the process.
+
+
+### Commenter
+
+Community members are welcome to join in on the discussion. Commenters may ask
+clarifying questions, provide additional use cases, identify edge cases, etc.
+
+
+## When is a PEP required?
+
+A PEP is helpful when we need to have an agreed upon design, analysis of
+risk/scope, or evaluation of user experience up-front. Based on that a PEP is
+required when altering:
+
+* CLI syntax
+
+* Configuration syntax (such as the porter.yaml or config.toml)
+
+* Communication between Porter and mixins/plugins
+
+* Essential behavior of Porter (such as defaults or anything that significantly
+  impacts what a mixin developer, bundle author or end-user needs to know)
+
+In general it isn't necessary for bug fixes, documentation updates, website
+tweaks, routine build upkeep or [meta] project concerns. For example, correcting
+a typo in the PEP documentation can be made directly with a pull request, however
+adding a new requirement to the process would require a new PEP. A maintainer
+may require a PEP when they feel that a change would benefit from the PEP
+process.
+
+
+## PEP Contents
+
+Proposals are stored in this repository in the [pep](/pep/) directory. They
+should be named `NNN-title.md` where `NNN` is the PEP number and `title` is the
+lower-case title of the PEP with hyphens in place of spaces. For example, the
+filename for this PEP is `001-initial-pep-process.md`.
+
+Every PEP should have the following sections:
+
+1. [Preamble](#preamble)
+1. [Abstract](#abstract)
+1. [Motivation](#motivation)
+1. [Rationale](#rationale)
+1. [Specification](#specification)
+1. [Backwards Compatibility](#backwards-compatibility)
+1. [Security Implications](#security-implications)
+1. [Rejected Ideas](#rejected-ideas)
+1. [Open Questions](#open-questions)
+
+
+### Preamble
+
+Metadata about the PEP, including:
+
+* A short descriptive title
+* Number
+* Type
+* Status
+* The GitHub usernames of the author(s)
+* The GitHub username of the owner
+* The date the PEP was created
+
+
+### Abstract
+
+A short (~200 word) description of the technical issue being addressed.
+
+
+### Motivation
+
+It should clearly explain why Porter's existing functionality is inadequate to
+address the problem that the PEP solves and identify the impacted audience(s) (mixin
+developers, bundle authors, end-users). PEP submissions without sufficient
+motivation may be rejected outright. This is the most important part at the
+beginning and is required before moving forward.
+
+
+### Rationale
+
+The rationale fleshes out the specification by describing why particular design
+decisions were made.  It should describe alternate designs that were considered
+and related work.
+
+The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during discussion.
```

Comment:
nit: this appears to be an outlier line that isn't broken up at/by the 80th (?) column number